app/lib/features/chat/widgets/messages/location_message.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_chat_types/flutter_chat_types.dart';
+
+class LocationMessageWidget extends StatelessWidget {
```

Comment:
Ahh, I see. We have support of the location message as well. I never used/see this in acter.

Reply:
yeah, dunno if we ever implemented sending it...